src/ducks/myApps/index.js

```diff
@@ -8,29 +8,37 @@ import {
   NotUninstallableAppException
 } from '../../lib/exceptions'
 
-const FETCH_MY_APPS = 'FETCH_MY_APPS'
-const FETCH_MY_APPS_SUCCESS = 'FETCH_MY_APPS_SUCCESS'
-const FETCH_MY_APPS_FAILURE = 'FETCH_MY_APPS_FAILURE'
+const NOT_REMOVABLE_APPS = ['drive', 'collect']
```

Comment:
Aren't we supposed to get those kind of information from the stack ? I suggest to add a comment to remember talking to them about that point.

Reply:
I don't think so. These apps can't be removed because they are a necessary part of the Cozy platform itself and not due to a specific context. But we could discuss about that.